Dockerfile.bionic

```diff
@@ -31,11 +31,5 @@ COPY scripts/install_abseil.sh cartographer/scripts/
 RUN cartographer/scripts/install_abseil.sh && rm -rf /var/lib/apt/lists/*
 COPY scripts/install_proto3.sh cartographer/scripts/
 RUN cartographer/scripts/install_proto3.sh && rm -rf protobuf
-COPY scripts/install_grpc.sh cartographer/scripts/
-RUN cartographer/scripts/install_grpc.sh && rm -rf grpc
-COPY scripts/install_async_grpc.sh cartographer/scripts/
-RUN cartographer/scripts/install_async_grpc.sh && rm -rf async_grpc
-COPY scripts/install_prometheus_cpp.sh cartographer/scripts/
-RUN cartographer/scripts/install_prometheus_cpp.sh && rm -rf prometheus-cpp
```

Comment:
Do you know which one of these is broken?

Reply:
Yes, thanks for asking. I now wrote down my results and possible approach in #1894.